### PR TITLE
refactor: add separate accounts  for each policy in e2e tests

### PR DESCRIPTION
**Pull request by kingpinXD**
# Description

Closes https://github.com/zeta-chain/node/issues/2447
# How Has This Been Tested?

- [ ] Tested CCTX in localnet
- [ ] Tested in development environment
- [ ] Go unit tests
- [ ] Go integration tests
- [ ] Tested via GitHub Actions

## Summary by CodeRabbit

- **New Features**
  - Introduced separate policy accounts: emergency, operational, and admin, for more granular control.

- **Refactor**
  - Replaced `FungibleAdminName` with new policy names across tests and configurations.
  - Updated scripts to support newly defined policy accounts.
  - Enhanced end-to-end test configuration to utilize multiple policy accounts.

- **Bug Fixes**
  - Fixed issues with transaction broadcasting by using specific policy accounts.

- **Documentation**
  - Updated changelog to reflect changes in account handling and policy account introductions.

### CODE REVIEW

File: e2e/config/config.go [hunk 1]
```diff
@@ -257,7 +282,15 @@
...
-	c.AdditionalAccounts.UserFungibleAdmin, err = generateAccount()
+	c.PolicyAccounts.EmergencyPolicyAccount, err = generateAccount()
+	if err != nil {
+		return err
+	}
+	c.PolicyAccounts.OperationalPolicyAccount, err = generateAccount()
+	if err != nil {
+		return err
+	}
+	c.PolicyAccounts.AdminPolicyAccount, err = generateAccount()
```
[the remark this answers is not in the document]
Comment (lumtis) [reply e2e/config/config.go:293]: @coderabbitai we don't add test in `e2e` package

**Comment by lumtis** [comment]
Need https://github.com/zeta-chain/node/pull/2463 to be merged to fully test the functionality

**Review by lumtis — APPROVED** [review]
LGTM

File: e2e/config/config.go [hunk 2]
```diff
@@ -216,9 +229,21 @@ func (c Config) Validate() error {
...
 		err := account.Validate()
 		if err != nil {
-			return fmt.Errorf("validating account %d: %w", i, err)
+			return fmt.Errorf("validating additional account %d: %w", i, err)
 		}
 	}
+
+	policyAccounts := c.PolicyAccounts.AsSlice()
+	for i, account := range policyAccounts {
+		if account.RawEVMAddress == "" {
```
[the remark this answers is not in the document]
Comment (kingpinXD) [reply e2e/config/config.go:238]: I dont think we need to have all the additional and policy accounts available everytime .

**Pull request merged**